### PR TITLE
hammer: build/ops: fix logrotate on debian-based distros.

### PR DESCRIPTION
**Pull request by Abhishekvrshny**
Fixes: http://tracker.ceph.com/issues/19190

Signed-off-by: Abhishek Varshney <abhishek.varshney@flipkart.com>

### CODE REVIEW

**Comment by smithfarm** [comment]
Hi @Abhishekvrshny - can this be done as a cherry-pick from master?

**Comment by Abhishekvrshny** [comment]
@smithfarm : `src/rgw/logrotate.conf` no longer exists on master. Also, this fix is no longer needed for infernalis and newer releases, as systemd support for debian already exists for them.

**Comment by smithfarm** [comment]
Great, can you add this explanation to the commit message, please? I.e. something like "This fix cannot be cherry-picked from master because . . ."

**Comment by Abhishekvrshny** [comment]
@smithfarm : done

File: src/rgw/logrotate.conf [hunk 1]
```diff
@@ -5,7 +5,14 @@
     sharedscripts
     postrotate
         if which service > /dev/null 2>&1 && [ -x `which service` ]; then
-            service ceph-radosgw reload >/dev/null
+            case $(lsb_release -si) in
```
Comment (amitkumar50) [inline src/rgw/logrotate.conf]: @Abhishekvrshny (NIT) Though close, Please consider to use title < 50 characters for PRs.
https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#title-of-pull-requests-and-title-of-commits

**Comment by smithfarm** [comment]
Hammer is EOL.

**Pull request closed without merging**